### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by jklukas**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,9 +1,11 @@
-redshift_sqlalchemy
+sqlalchemy-redshift
 ===================
 
 Amazon Redshift dialect for SQLAlchemy.
 
 .. image:: https://travis-ci.org/sqlalchemy-redshift/sqlalchemy-redshift.png?branch=master
+   :target: https://travis-ci.org/sqlalchemy-redshift/sqlalchemy-redshift
+   :alt: Travis CI build status
```
Comment (jklukas) [inline README.rst]: This fixes the current problem where the image fails to load on rtfd.

File: README.rst [hunk 2]
```diff
@@ -20,5 +22,6 @@ The DSN format is similar to that of regular Postgres::
     >>> sa.create_engine('redshift+psycopg2://username@host.amazonaws.com:5439/database')
     Engine(redshift+psycopg2://username@host.amazonaws.com:5439/database)
 
-See the docstring for `RedshiftDDLCompiler` in
-`dialect.py <redshift_sqlalchemy/dialect.py>`_ for more detail.
+See the `RedshiftDDLCompiler documentation
+<http://sqlalchemy-redshift.readthedocs.org/en/latest/ddl-compiler.html>`_
+for details on Redshift-specific features the dialect supports.
```
Comment (jklukas) [inline README.rst]: Likewise, this link is currently broken on rtfd. This version should work on both GitHub and rtfd.

Comment (graingert) [inline README.rst]: https://sqlalchemy-redshift.readthedocs.org/en/latest/ddl-compiler.html

File: README.rst (hunk 1 above)
Comment (graingert) [inline README.rst]: nice

**Comment by graingert** [comment]
Nice, make sure the link is HTTPS and you can merge this

**Comment by jklukas** [comment]
Updated link to https.

**Pull request merged**